4.x/typescript/Recenter/app/Recenter.tsx

```diff
@@ -32,17 +30,15 @@ interface RecenterParams extends __esri.WidgetProperties {
   initialCenter: number[]
 }
 
-
 @subclass("esri.widgets.Recenter")
 class Recenter extends Widget {
   constructor(params?: RecenterParams) {
     super(params);
-    this._onViewChange = this._onViewChange.bind(this);
+    // this._onViewChange = this._onViewChange.bind(this);
```

Comment:
Remove constructor since it's not needed.

Reply:
`index.html` demonstrates using RecenterParams in the constructor. I'm deleting the commented out line.